libraries/ui-strings/src/main/res/values/localazy.xml

```diff
@@ -129,6 +129,8 @@
   <string name="action_view_source">"View source"</string>
   <string name="action_yes">"Yes"</string>
   <string name="action_yes_try_again">"Yes, try again"</string>
+  <string name="banner_migrate_to_native_sliding_sync_description">"Your server now supports a new, faster protocol. Log out and log back in to upgrade now. Doing this now will help you avoid a forced logout when the old protocol is removed later."</string>
+  <string name="banner_migrate_to_native_sliding_sync_title">"Upgrade available"</string>
```

Comment:
Not used anymore, I guess those 2 strings will be removed soon form Localazy. We should just not forget.

Reply:
Maybe we should deprecate them first. It's very unlikely that we'll have to create some version where this strings still exist (hotfix?), but maybe we should keep them for a while.